run.py

```diff
@@ -24,33 +24,64 @@ def str_to_bool(s):
     parser_prepare.set_defaults(run=manager.prepare)
     parser_prepare.add_argument("--repo", required=True, type=str, help="the image repo (i.e., osirrc2019/anserini)")
     parser_prepare.add_argument("--tag", default="latest", type=str, help="the image tag (i.e., latest)")
-    parser_prepare.add_argument("--save_to_snapshot", default="save", type=str, help="used to determine the tag of the snapshotted image after indexing")
+    parser_prepare.add_argument("--save_to_snapshot", default="save", type=str,
+                                help="used to determine the tag of the snapshotted image after indexing")
     parser_prepare.add_argument("--collections", required=True, nargs="+", help="the name of the collection")
     parser_prepare.add_argument("--opts", nargs="+", default="", type=str, help="the args passed to the index script")
     parser_prepare.add_argument("--version", default="", type=str, help="the version string passed to the init script")
 
+    # Specific to train
+    trainer_prepare = parser_sub.add_parser("train")
+    trainer_prepare.set_defaults(run=manager.train)
+    trainer_prepare.add_argument("--repo", required=True, type=str, help="the image repo (i.e., albep/nvsm)")
+    trainer_prepare.add_argument("--tag", default="latest", type=str, help="the image tag (i.e., latest)")
+    trainer_prepare.add_argument("--load_from_snapshot", default="save", type=str,
+                                 help="used to determine the tag of the snapshotted image to interact with")
```

Comment:
Instead of "to interact with", which is specific to the `interactor` module, this should probably be something more like "to train from".

Reply:
Good observation! I'll change that in the next commit.